mesa/agent.py

```diff
@@ -264,17 +264,24 @@ def do(
 
         return res if return_results else self
 
-    def get(self, attr_name: str) -> list[Any]:
+    def get(self, attr_names: str | list[str]) -> list[Any]:
         """
-        Retrieve a specified attribute from each agent in the AgentSet.
+        Retrieve the specified attribute(s) from each agent in the AgentSet.
 
         Args:
-            attr_name (str): The name of the attribute to retrieve from each agent.
+            attr_names (str | List[str]): The name(s) of the attribute(s) to retrieve from each agent.
 
         Returns:
-            list[Any]: A list of attribute values from each agent in the set.
+            list[Any]: A list of attribute values for each agent in the set.
```

Comment:
Is this still correct? Would it be nested in some sort when a list of attrs is inputted?

Maybe make a little bit more explicit what is returned when a str is inputted for attr_names, and what when a list of strings is inputted.

Reply:
see the updated text